Streetcode/Streetcode.WebApi/Utils/WebParsingUtils.cs

```diff
@@ -325,6 +325,10 @@ public static string OptimizeStreetname(string streetname)
             }
         }
 
+        public static async Task<bool> SaveToponymToDb(AddressEntity addressEntity)
```

Comment:
- No **Async** suffix even though method is asynchronous;
- Method can be made private

Reply:
1- done, 2-someday you will need it public :)

Reply:
Ok, makes sense.